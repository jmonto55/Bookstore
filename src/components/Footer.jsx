import styles from '../sass/Footer.module.scss';

const Footer = () => (
  <footer className={styles.footer} data-testid="footer">
    <p>Â© 2023 Jose Montoya. All rights reserved.</p>
    <p>Built with ðŸ’™ in Medellin, CO</p>
  </footer>
);

export default Footer;
