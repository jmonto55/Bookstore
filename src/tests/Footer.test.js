import { render, screen } from '@testing-library/react';
import Footer from '../components/Footer';
import '@testing-library/jest-dom/extend-expect';

describe('Test suite for the Footer component', () => {
  let footer;
  beforeEach(() => {
    render(<Footer />);
    footer = screen.getByTestId('footer');
  });

  it('Should test if Footer component is inside the document', () => {
    expect(footer).toBeInTheDocument();
  });

  it('Should render a snapshot of the Footer component', () => {
    expect(footer).toMatchSnapshot();
  });
});
