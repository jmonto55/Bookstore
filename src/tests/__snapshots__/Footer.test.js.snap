// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for the Footer component Should render a snapshot of the Footer component 1`] = `
<footer
  class="footer"
  data-testid="footer"
>
  <p>
    Â© 2023 Jose Montoya. All rights reserved.
  </p>
  <p>
    Built with ðŸ’™ in Medellin, CO
  </p>
</footer>
`;
